### PR TITLE
Modjoin: check if on/off before proceeding

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -557,6 +558,7 @@ exports.commands = {
 			}
 		} else {
 			if (target === 'on' || target === 'true') {
+				if (room.modjoin) return this.errorReply("Modjoin is already turned on in this room.");
```
Comment (sirDonovan) [inline commands.js]: <code>room.modjoin</code> can be truthy if set to a specific rank so you should check <code>=== true</code> here. You could also check for setting to the same rank below.

Comment (panpawn) [inline commands.js]: How is that a different check though?

Also ya sure, I'll update this with that when I wake up.

Comment (panpawn) [inline commands.js]: Oh nevermind, I think I know what you mean now.

**Comment by panpawn** [comment]
Good catch, @sirDonovan! Thanks!

**Comment by panpawn** [comment]
Sidenote of discussion: I think just using ``/modjoin`` should be like just using ``/modchat``, and it should tell you what rank modjoin is currently set to like modchat does.  Thoughts?

**Comment by Zarel** [comment]
I agree, @panpawn

**Pull request merged**